CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changelog
 ### Improvements and Changes
 
 -   Type hints have been added to the `PauliTerm` class (@rht, gh-1075).
+-   `QAM.write_memory` now accepts either a Sequence of values or a single value. (@tommy-moffat, gh-658)
```

Comment:
```suggestion
-   `QAM.write_memory` now accepts either a `Sequence` of values or a single value
    (@tommy-moffat, gh-1079).
```

Reply:
You can just click the "Commit suggestion" and it handles it for you. Note that I changed the `gh-###` to the PR number rather than the issue number.

Reply:
I was able to resolve this myself, but the other suggestion still applies!